misk-crypto/src/main/kotlin/misk/crypto/Cipher.kt

```diff
@@ -31,23 +33,23 @@ interface Cipher {
    * This includes the algorithm type, key identifiers, primary identifier
    * and a Base64 encoded and encrypted key
    */
-  val keyInfo: KeyInfo
+  val keyInfo: List<KeyInfo>
 }
 
 /**
  * Real implementation of the [Cipher] interface.
  * This class uses key [KeysetHandle] that's been decrypted by the KMS when the module was loaded.
  */
 class RealCipher internal constructor(
-  private val keysetHandle: KeysetHandle,
-  private val masterKey: Aead
+  private val keys: List<Pair<KeysetHandle, Aead>>
```

Comment:
Maybe this should be `List<SomeDataClass>` instead

Reply:
Agree, I usually don't like using `Pair`, but Kotlin is really accommodating with these objects (w/ stuff like `{ (left, right) -> ... }`) and I got tempted.

I'll switch it to something more readable.